### PR TITLE
Feat/icon size copy text

### PR DESCRIPTION
**Pull request by rayyan224**
icon size copy text

### CODE REVIEW

File: src/components/Typography/Typography.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ const DynamicText = ({
     monospace,
     copyable,
     children,
+    iconSize = 22,
```
Comment (Y0moo) [inline src/components/Typography/Typography.tsx:35]: default Icon size for CopyButton is 1em, which means it takes the max height of the parent element. Why do we hardcode here `22`?

Comment (rayyan224) [reply src/components/Typography/Typography.tsx]: Did'nt see the 1em, good catch bro

**Review by BillyG83 — APPROVED** [review]
ok nice

**Review by BillyG83 — APPROVED** [review]
Niiiiice guys 🤩

**Pull request merged**